<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Doubler</class>
 <widget class="QMainWindow" name="Form_Doubler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doubler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>+1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Multiply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>x2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Reset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_current_value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Текущее значение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_final_value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Конечное значение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_steps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Текущее кол-во шагов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_minimal_steps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Минимальное кол-во шагов:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_minimal_steps_number">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_steps_number">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_final_value_number">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_current_value_number">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuNew_Game">
     <property name="title">
      <string>New Game</string>
     </property>
     <addaction name="actionEasy"/>
     <addaction name="actionNormal"/>
     <addaction name="actionHard"/>
    </widget>
    <addaction name="menuNew_Game"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
